doc/recipes.md

```diff
@@ -10,32 +10,41 @@ recipes
   |- eu.mikelangelo-project.osv.nfs  # <--- recipe
       |- build.sh
       |- demo     # <--- optional recipe test
-          |- pkg  # <--- directory containing demo package that requires the package that recipe is for
-          |   |- meta
-          |      |- package.yaml.templ
-          |      |- run.yaml
-          |- expected-stdout.txt  # <--- when demo package is run we expect following text in console
+      |   |- pkg  # <--- directory containing demo package that requires the package that recipe is for
+      |   |   |- meta
+      |   |      |- package.yaml.templ
+      |   |      |- run.yaml
+      |   |- expected-stdout.txt  # <--- when demo package is run we expect following text in console
+      |- meta     # <--- content of this directory is copied as-it-is into package's meta/ directory
+          |- README.md
+          |- run.yaml
 ```
 
 ### Writing `build.sh` script
 The `build.sh` script is responsible for providing all the content of the package, including
-`meta/package.yaml` and optionally `meta/run.yaml`. Building is assumed successful when script exits
-with 0 status code or failed when any other exit code is returned.
+`meta/package.yaml`. Building is assumed successful when script exits with 0 status code or failed
+when any other exit code is returned.
 
 Container's main process (implemented in [capstan-packages.py](../docker_files/capstan-packages.py))
 runs the `build.sh` script from within the recipe directory and provides following environment
 variables to it:
 
 | ENV | EXAMPLE VALUE | DESCRIPTION |
 |-----|---------------|-------------|
-| `RECIPE_DIR` | /recipes/eu.mikelangelo-project.osv.nfs | recipe directory |
-| `PACKAGE_RESULT_DIR` | /results/eu.mikelangelo-project.osv.nfs | directory where build result must appear |
-| `PACKAGE_NAME` | eu.mikelangelo-project.osv.nfs | same as directory name of recipe |
+| `RECIPE_DIR` | /recipes/osv.nfs | recipe directory |
```

Comment:
Are these variables ordered in some sort? I believer the first two are most important, however a lexicographical order would be more natural now that there are many.